.teamcity/settings.kts

```diff
@@ -217,15 +217,18 @@ object RunAllUnitTests : BuildType({
 		script {
 			name = "Prepare environment"
 			scriptContent = """
-				set -e
+				#!/bin/bash
```

Comment:
#!/bin/bash allows us to use bash syntax in this script without fear of it running in a different shell.

The `set` lines come from https://kvz.io/bash-best-practices.html